src/components/Header/Nav/Nav.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Nav.css';
+
+function Nav() {
```

Comment:
Use full component names
```suggestion
function Navigation() {
```

Reply:
but tag named `nav`?